### PR TITLE
Feature/#715 btn toggle lbls issue

### PR DESCRIPTION
**Pull request by maria-vslvn**
1. issue with active/non-active state of toggle btn was fixed

### CODE REVIEW

File: src/components/auction/AuctionBody/index.tsx [hunk 1]
```diff
@@ -11,12 +11,14 @@ import { OrderBookContainer } from '../OrderbookContainer'
 import OrdersTable from '../OrdersTable'
 
 const Grid = styled.div`
-  display: grid;
+  display: flex;
+  flex-direction: column;
   grid-template-columns: 1fr;
```
Comment (ramirotw) [inline src/components/auction/AuctionBody/index.tsx:16]: is it `grid-template-columns` valid in a flex container?

Comment (maria-vslvn) [reply src/components/auction/AuctionBody/index.tsx:16]: extra css - fixed

**Comment by josojo** [comment]
LGTM

A good habit is to always start new PRs form the latest develop environment and writing "closes: #715" into the PR, as this will automatically close your PR once the PR gets merged

**Comment by elena-zh** [comment]
Hi @maria-vslvn !
Great, now the toggle works as expected.
However, the issue with 'empty' orderbook list value is not completely fixed: I am able to see it, but only after I press on the Granularity dropdown. 

https://user-images.githubusercontent.com/70885163/123759230-6127f700-d8c8-11eb-9595-5c747bab7c40.mp4


Could you please fix it?

Thanks

**Comment by maria-vslvn** [comment]
@elena-zh fix added

**Pull request merged**